src/components/sound.js

```diff
@@ -24,44 +23,33 @@ module.exports.Component = registerComponent('sound', {
 
   update: function (oldData) {
     var data = this.data;
-    var diffData = diff(oldData || {}, data);
     var el = this.el;
     var sound = this.sound;
-    var src = data.src;
-    var srcChanged = 'src' in diffData;
+    var srcChanged;
+
+    oldData = oldData || {};
```

Comment:
Yeah, by flag, I meant use `init` and store a variable.

Actually, you already changed it in the refactor PR. It's an empty object at the beginning now. I don't have to do anything here.